native-src/android/app/src/main/java/com/telerik/localnotifications/Builder.java

```diff
@@ -84,6 +94,17 @@ static Notification build(JSONObject options, Context context, int notificationI
 
     // Notification styles:
 
+    private static void applyNotificationLed(JSONObject options, NotificationCompat.Builder builder) {
+        if (shouldEnableNotificationLed(options)) {
+            try {
+                JSONObject notificationLed = options.getJSONObject("notificationLed");
+                builder.setLights(notificationLed.optInt("ledColor", DEFAULT_NOTIFICATION_COLOR), notificationLed.optInt("ledOn", DEFAULT_NOTIFICATION_LED_ON), notificationLed.optInt("ledOff", DEFAULT_NOTIFICATION_LED_OFF));
```

Comment:
Apologies but I am actually quite new - may I ask if this (combined with [this `color=color.android`](https://github.com/EddyVerbruggen/nativescript-local-notifications/pull/133/files#diff-75678843cd805744830e5ddc817a4a44R219)) the correct way to use `Color`?